### DISCUSSION TITLE
Can the check result be changed using any command?

### DISCUSSION THREAD

**Discussion by mridang** [community/community#26533] [Actions]
<p>When the checks run, the checks overview show a simple status of the check result i.e. whether it passed or failed.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/8fa0b578777d2196af77cd0a7f464f63bf1d261d.png" data-download-href="/uploads/short-url/kuAAURMy5l3Dbz1D8NZCDWFwM5n.png?dl=1" title="check_run_conclusion" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/327432/181098211-721f8c1e-134d-4c2a-9e5a-d4d84f3c8711.png" alt="check_run_conclusion" data-base62-sha1="kuAAURMy5l3Dbz1D8NZCDWFwM5n" width="690" height="408" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/8fa0b578777d2196af77cd0a7f464f63bf1d261d_2_690x408.png, https://aws1.discourse-cdn.com/github/optimized/2X/8/8fa0b578777d2196af77cd0a7f464f63bf1d261d_2_1035x612.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/8/8fa0b578777d2196af77cd0a7f464f63bf1d261d_2_1380x816.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/8fa0b578777d2196af77cd0a7f464f63bf1d261d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">check_run_conclusion</span><span class="informations">1778×1052 123 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Can this be changed without making any API calls? I’ve seen this happen for checks that leverage the API but I haven’t seen this for simple scripted actions.</p>
<p>For example, along with the status - it would be handy to be able to show a message when linting, that reads “30 linter warnings and 200 errors”.</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi <a class="mention" href="/mridang">@mridang</a> ,</p>
<p>There are not any command to change status check result. The javascript actions also use Github API to make changes to status check. There is an action to update commit status: <a href="https://github.com/ouzi-dev/commit-status-updater" rel="nofollow noopener">https://github.com/ouzi-dev/commit-status-updater</a></p>
<p>I am afraid that I have to say API calls is the final method. </p>
